crates/swc/tests/tsc-references/importCallExpressionInExportEqualsAMD_es2015.1.normal.js

```diff
@@ -1,9 +1,5 @@
-import _async_to_generator from "@swc/helpers/src/_async_to_generator.mjs";
-// @filename: index.ts
-module.exports = _async_to_generator(function*() {
-    const something = yield import("./something");
-});
 // @module: amd
 // @target: esnext
 // @filename: something.ts
```

Comment:
There are a lot of confusing changes here.
Since swc test kit cannot handle the `@filename` now, the contents of multiple files are treated as one file.